### PR TITLE
[DOCFIX] Improve some sub-header in secure HDFS doc to highlight

### PR DESCRIPTION
**Pull request by ifcharming**
important configurations setup

### CODE REVIEW

**Comment by ifcharming** [comment]
@apc999

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 1]
```diff
@@ -50,18 +48,22 @@ For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_H
...
-Alternatively, you can also create the configuration file from the template and set the contents manually. 
+Alternatively, you can also create the configuration file from the template and set the contents manually.
 
 {% include Common-Commands/copy-alluxio-env.md %}
 
-Then edit `alluxio-site.properties` file to set the under storage address to the HDFS namenode address
-(e.g., `hdfs://localhost:9000` if you are running the HDFS namenode locally with default port).
+Then edit `conf/alluxio-site.properties` file to set the under storage address to the HDFS namenode address
+and the HDFS directory you want to mount to Alluxio. For example, the under storage address can be
+`hdfs://localhost:9000` if you are running the HDFS namenode locally with default port and mapping HDFS root directory to Alluxio,
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: "mapping" -> "mounting"

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 2]
```diff
@@ -50,18 +48,22 @@ For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_H
...
+or `hdfs://localhost:9000/alluxio/data` if only the HDFS directory `/alluxio/data` is mapped to Alluxio.
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: "mapped" -> "mounted"

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 3]
```diff
@@ -50,18 +48,22 @@ For example, if you are running Alluxio on your local machine, `ALLUXIO_MASTER_H
...
 
 {% include Configuring-Alluxio-with-HDFS/underfs-address.md %}
 
-Copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
+## HDFS configuration files 
+Please copy secure HDFS conf xml files (`core-site.xml`, `hdfs-site.xml`, `mapred-site.xml`, `yarn-site.xml`) to
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: "conf" -> "configuration" ?

Also, could you add a little bit as to why we should copy those files?

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md [hunk 4]
```diff
@@ -89,6 +92,7 @@ details about setting configuration parameters can be found in
 
 # Running Alluxio Locally with secure HDFS
 
+Before this step, please make sure your HDFS cluster is running and the directory mapped to Alluxio exists.
```
Comment (gpang) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: "mapped" -> "mounted"

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 1 above)
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: done

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 2 above)
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: done

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 3 above)
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: done, thanks.

File: docs/en/Configuring-Alluxio-with-secure-HDFS.md (hunk 4 above)
Comment (ifcharming) [inline docs/en/Configuring-Alluxio-with-secure-HDFS.md]: done

**Comment by ifcharming** [comment]
@gpang  thanks for the comments. PTAL

**Pull request merged**